coverage/.last_run.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
We should probably remove all coverage files from github in gitignore

Reply:
Good catch!